cspell.json

```diff
@@ -30,6 +30,7 @@
 		"isbot",
 		"Jérémie",
 		"knip",
+		"Neurodiversity",
```

Comment:
I don't think you need this because CSpell didn't flag it and it exists in their dictionaries: https://github.com/search?q=repo%3Astreetsidesoftware%2Fcspell-dicts%20neurodiversity&type=code

```suggestion
```